src/pkg/repository/repository.go

```diff
@@ -237,6 +184,11 @@ func (r *repository) Connect(ctx context.Context) (err error) {
 		}
 	}()
 
+	// ctrl, err = connectToM365(ctx, sel.PathService(), r.Account, r.Opts)
```

Comment:
Is this commented out on purpose?

Reply:
Oh!  Heh, yes.  Was supposed to be deleted.  But it'll get patched up in the following PR either way.  Good call out.